commands.go

```diff
@@ -248,12 +248,27 @@ type Cmdable interface {
 
 	BZPopMax(ctx context.Context, timeout time.Duration, keys ...string) *ZWithKeyCmd
 	BZPopMin(ctx context.Context, timeout time.Duration, keys ...string) *ZWithKeyCmd
+
+	// TODO: remove
+	// 		ZAddNX
+	//		ZAddXX
```

Comment:
I would keep these 2 in v9. They looks like a popular option.

Reply:
OK, I will make adjustments